appservice/src/editScmType.ts

```diff
@@ -12,9 +12,10 @@ import { ScmType } from './ScmType';
 import { SiteClient } from './SiteClient';
 import { nonNullProp } from './utils/nonNull';
 
-export async function editScmType(client: SiteClient, node: AzureTreeItem): Promise<string | undefined> {
+export async function editScmType(client: SiteClient, node: AzureTreeItem, newScmType?: ScmType): Promise<string | undefined> {
     const config: WebSiteManagementModels.SiteConfigResource = await client.getSiteConfig();
-    const newScmType: string = await showScmPrompt(nonNullProp(config, 'scmType'));
+    // tslint:disable-next-line:strict-boolean-expressions
+    newScmType = newScmType ? newScmType : await showScmPrompt(nonNullProp(config, 'scmType'));
```

Comment:
"ERROR: (strict-boolean-expressions) C:/Users/naturins/Desktop/Code/vscode-azuretools/appservice/src/editScmType.ts[17, 18]: This type is not allowed in the condition because it could be an enum. Allowed types are boolean, null-union, undefined-union, or string."

I think because I'm not checking a string, but an enum, it's giving me this error now.

Reply:
Okey dokey